File: src/main/java/com/box/Box.java
package com.box;

public class Box {
    float length;
    float width;
    float height;
    String name;

    public boolean validate(float length,float width, float height){
        return length <= this.length && width <= this.width && height <= this.height ;
    }

    public String getName() {
        return name;
    }
}

File: src/main/kotlin/com/box/Box.kt
package com.box.kotlin

 open class Box(var length: Float,var width: Float,var height: Float){

     fun validate(boxSize: Map<String, Float>): Boolean = validate(boxSize.get("length")!!, boxSize.get("width")!!, boxSize.get("height")!!)
     fun validate(length: Float, width: Float, height: Float): Boolean = length <= this.length && width <= this.width && height <= this.height
}

